### PR TITLE
Hkm wip frd automation

### PR DESCRIPTION
**Pull request by hari66mech**
Please Review the code.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -36,17 +36,12 @@
         </plugins>
     </build>
     <dependencies>
-<<<<<<< HEAD
```
Comment (rajmyil) [inline pom.xml:39]: Why the conflicts are not resolved?

File: src/main/java/com/frd/automation/testscripts/FalconDemoTestReportTest.java [hunk 2]
```diff
@@ -19,13 +25,31 @@
...
+        ExcelDataReader.renameDownloadedFile();
+        Thread.sleep(Constants.TIMEOUT_SECONDS);
+    }
 
     /**
      * This is the reports Dashboard Validations method Which is verify Dashboard url and Project Name.
      *
      * @throws InterruptedException
      */
-    @Test
```
Comment (rajmyil) [inline src/main/java/com/frd/automation/testscripts/FalconDemoTestReportTest.java:28]: @Test listed twice

File: src/main/resources/locators/falcondemo.properties [hunk 3]
```diff
@@ -2,4 +2,6 @@ test.report.sheet=Test Report
 test.cases.sheet=Test Cases
 dashboards.url=http://10.10.10.231:8082/#/app/dashboard
 project.title=FalconDemo
-dashboard.title=ATMECS - Falcon Test Repotrs
```
Comment (rajmyil) [inline src/main/resources/locators/falcondemo.properties:5]: Line no 5 listed twice

**Review by rajmyil — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (hari66mech) [reply pom.xml:39]: Resolved

File: pom.xml [hunk 4]
```diff
@@ -36,17 +36,12 @@
...
-=======
```
[the remark this answers is not in the document]
Comment (hari66mech) [reply pom.xml:40]: Resolved.

File: pom.xml [hunk 5]
```diff
@@ -56,23 +51,11 @@
         </dependency>
 
         <!-- https://mvnrepository.com/artifact/org.testng/testng -->
->>>>>>> 554043ccce03c12b1e0fdf90aadce467ae613fb5
```
[the remark this answers is not in the document]
Comment (hari66mech) [reply pom.xml:59]: Resolved.

File: src/main/java/com/frd/automation/constants/Constants.java [hunk 6]
```diff
@@ -49,7 +46,7 @@
     public final static String CONFIG_FILE = RESOURCES_HOME + "config.properties";
 
     public final static String LOCATOR_HOME = RESOURCES_HOME + "locators" + File.separator;
-    public final static String TESTDATA_HOME = RESOURCES_HOME + "testdata" + File.separator;
```
[the remark this answers is not in the document]
Comment (hari66mech) [reply src/main/java/com/frd/automation/constants/Constants.java:52]: Resolved.

File: src/main/java/com/frd/automation/testscripts/FalconDemoTestReportTest.java (hunk 2 above)
Comment (hari66mech) [reply src/main/java/com/frd/automation/testscripts/FalconDemoTestReportTest.java:28]: Resolved.

File: src/main/resources/locators/falcondemo.properties (hunk 3 above)
Comment (hari66mech) [reply src/main/resources/locators/falcondemo.properties:5]: Resolved.

**Pull request merged**